test/framework/batch_helper.hpp

```diff
@@ -197,7 +197,10 @@ namespace framework {
       auto result_batch = shared_model::interface::TransactionBatchFactory::
           createTransactionBatch(txs, TxsValidator());
 
-      return framework::expected::val(result_batch).value().value;
+      return boost::get<iroha::expected::Value<
```

Comment:
Is it possible to overload expected::val for unique_ptr? Or Result itself require type to be copy-able?

Reply:
I noticed some changes for expected::val/err but this line remains the same. Could you explain this?